lib/ngx/errlog.md

```diff
@@ -210,6 +211,30 @@ clear the table yourself before feeding it into the `errlog.get_logs` function.
 
 [Back to TOC](#table-of-contents)
 
+get_sys_filtering_level
+-----------------------
+**syntax:** *log_level = log_module.get_sys_filtering_level()*
+
+**context:** *any*
+
+Return system default filtering level as an integer. Later the return value
+could be used as Nginx log level constant. For example:
+
+```lua
+init_by_lua_block {
```

Comment:
We can omit the directive here. Since it does not really matter which handler it is in. Besides, the code block marks  as the`lua` language instead of the `nginx` language.